ReleaseTooling/Sources/ZipBuilder/CarthageUtils.swift

```diff
@@ -18,6 +18,17 @@ import CommonCrypto
 import Foundation
 import Utils
 
+struct CarthageBuildOptions {
+  /// Location of directory containing all JSON Carthage manifests.
+  let jsonDir: URL
+
+  /// Version checking flag.
+  let isVersionCheckEnabled: Bool
+
+  /// An RC number, if it exists.
+  let rcNumber: Int?
```

Comment:
I'm not sure this is still needed since each build gets built into a time-stamped temp directory

Reply:
That's a good point - we originally had it in the filename as well so we could upload multiple versions and it'd be clear. Think it's worth keeping around until we figure out building on GHA or should I remove it and we can re-add if eneded?

Reply:
I don't have a strong opinion either way. Your call.

Reply:
Removed for now, will be easy to re-add.